holoviews/core/options.py

```diff
@@ -1748,7 +1748,7 @@ def id_offset(cls):
         max_ids = []
         for backend in Store.renderers.keys():
             store_ids = Store.custom_options(backend=backend).keys()
-            max_id = max(store_ids)+1 if len(store_ids) > 0 else 0
+            max_id = max(store_ids, default=0)+1
```

Comment:
I can't say why it didn't work before, but won't this change it to return 1 in the empty list case, where it used to return 0?

Reply:
Good point.

Reply:
Thankfully tests are just as attentive as you!